lib/drip/client/configuration.rb

```diff
@@ -6,7 +6,10 @@ class Configuration
       DEFAULT_URL_PREFIX = "https://api.getdrip.com/"
       private_constant :DEFAULT_URL_PREFIX
 
-      CONFIGURATION_FIELDS = %i[access_token api_key account_id url_prefix http_open_timeout http_timeout].freeze
+      CONFIGURATION_FIELDS = %i[
+        access_token api_key account_id url_prefix http_open_timeout
+        http_timeout skip_analytics
+      ].freeze
```

Comment:
I'd suggest going all the way and breaking each one into its own line so we can sort them too :). A lot of these list have been split up over the past couple years.